ex/high_throughput_inference/standalone_workermanager.py

```diff
@@ -58,17 +58,17 @@
 
 from smartsim._core.entrypoints.service import Service
 from smartsim._core.mli.comm.channel.channel import CommChannelBase
-from smartsim._core.mli.comm.channel.dragonchannel import DragonCommChannel
-from smartsim._core.mli.comm.channel.dragonfli import DragonFLIChannel
-from smartsim._core.mli.infrastructure.storage.dragonfeaturestore import (
+from smartsim._core.mli.comm.channel.dragon_channel import DragonCommChannel
```

Comment:
Should this file be standalone_worker_manager?

Reply:
Oops, yes! Fixed.